etcdserver/server.go

```diff
@@ -340,6 +344,7 @@ func NewServer(cfg *ServerConfig) (*EtcdServer, error) {
 		versionRt:     prt,
 		reqIDGen:      idutil.NewGenerator(uint8(id), time.Now()),
 		forceVersionC: make(chan struct{}),
+		msgSnapC:      make(chan raftpb.Message, 128),
```

Comment:
where do we get the size of buffer `128`? If arbitrary, we should briefly comment.